### ISSUE TITLE
Muted not working

### ISSUE THREAD

**Issue by ilu** [videojs/videojs-youtube#385]
I am having trouble starting videos as muted. The following set-up does not mute the video:

```
<video
    id="vid1"
    class="video-js vjs-default-skin"
    controls
    autoplay
    muted
    width="640" height="264"
    data-setup='{ "techOrder": ["youtube"], "sources": [{ "type": "video/youtube", "src": "https://www.youtube.com/watch?v=xjS6SftYQaQ"}] }'>
</video>
```

Defining `muted="muted", muted="true", data-setup='{"muted": true}'` etc don't seem to help, neither does enabling Youtube jsAPI.

Using videojs 5.10.2 and videojs-youtube 2.0.8. No errors in console. Tested with Chrome 51 beta, Firefox 47 beta and Safari 9.1 on OS X 10.11.4

For a live example see http://codepen.io/anon/pen/JXqdZJ

**Comment by gkatsev** [comment]
I'm assuming that it probably doesn't get picked up correctly by videojs-youtube to be passed through to youtube itself.
Does calling `player.muted(true);` and `player.muted(false);` toggle mute?

**Comment by ilu** [comment]
I can confirm that calling `player.muted(true); and player.muted(false);` does toggle mute. Is this the preferred way of muting and if so, is there a callback or an event to listen to when the player has been initialised?

**Comment by gkatsev** [comment]
The `muted` attribute definitely should work, that's a bug.
As a workaround, you can listen to the `ready` event and call `player.muted(true)` until it is fixed.

**Comment by gkatsev** [comment]
available in 2.2.0.
